services/ui-src/src/routes/ReviewSubmit/ReviewSubmit.test.tsx

```diff
@@ -92,25 +81,25 @@ describe("Test /mcpar/review-and-submit view", () => {
     expect(modalTitle).toBeVisible();
   });
 
-  test("should fire the submitForm function when confirming the modal", async () => {
-    render(reviewSubmitInitialView);
+  test("ReviewSubmit updates report status on submit confirmation", async () => {
+    render(ReviewSubmitComponent_InProgress);
     const reviewSubmitButton = screen.getByText("Submit MCPAR")!;
     await userEvent.click(reviewSubmitButton);
     const modalSubmitButton = screen.getByTestId("modal-submit-button")!;
     await userEvent.click(modalSubmitButton);
-    await expect(mockReportMethods.updateReport).toHaveBeenCalledTimes(1);
+    await expect(mockUpdateReport).toHaveBeenCalledTimes(1);
```

Comment:
... And change this to this
```suggestion
    await expect(mockUpdateReport.updateReport).toHaveBeenCalledTimes(1);
```

You can save yourself a few lines of code